spec/recipes/config_spec.rb

```diff
@@ -120,31 +134,34 @@
         mode: 0644,
         source: 'st2.conf.erb',
         variables: {
```

Comment:
If so, you should add a comment about that since whoever modifies this list next will just add it to the end since its not alphabetized